app/config/parameters.yml.dist

```diff
@@ -105,6 +105,7 @@ parameters:
     display_name_shifters: false
     use_card_reader_to_validate_shifts: false
     max_time_at_end_of_shift: 0
+    max_nb_of_past_cycles_to_display: 3
```

Comment:
Pas sur de comprendre pourquoi dans le twig, tu n'utilises pas cette variable ?

Reply:
cf `{% for cycle in -1..-3 %}` => https://github.com/elefan-grenoble/gestion-compte/pull/710/files#diff-10d011c81513c034d0fc0f8647febf520973c08e4381548d21716fc7c212843eR75

Reply:
j'ai corrigé